patches/server/0085-Implement-SpawnerPreSpawnEvent.patch

```diff
@@ -7,22 +7,17 @@ Added registered listeners check to avoid alloc if no plugins are
 listening.
 
 diff --git a/src/main/java/net/minecraft/server/MobSpawnerAbstract.java b/src/main/java/net/minecraft/server/MobSpawnerAbstract.java
-index e54f7edbd1d834391f6760ea7b82cf6af4ddf994..9607c55cdc98efe103e17a939a20d82ff66a7b02 100644
+index e54f7edbd1d834391f6760ea7b82cf6af4ddf994..b1780579735e025f469f86642bf303a9ccb9b207 100644
 --- a/src/main/java/net/minecraft/server/MobSpawnerAbstract.java
 +++ b/src/main/java/net/minecraft/server/MobSpawnerAbstract.java
-@@ -91,6 +91,17 @@ public abstract class MobSpawnerAbstract {
+@@ -91,6 +91,12 @@ public abstract class MobSpawnerAbstract {
                  boolean flag = false;
  
                  for (int i = 0; i < this.spawnCount; ++i) {
 +                    // PandaSpigot start - SpawnerPreSpawnEvent
-+                    if (net.techcable.tacospigot.event.entity.SpawnerPreSpawnEvent.getHandlerList().getRegisteredListeners().length > 0) {
```

Comment:
As you removed the patch that skipped events that had no registered listeners, you can re-add for this event

Reply:
If my mind is not failing you added the check back in however removed the use of "patches/api/0025-Add-a-call-helper-to-Event.patch" in the patch